modules/aws/state_manager/locals.tf

```diff
@@ -0,0 +1,3 @@
+locals {
+  state_management_key = "global/${var.environment}/s3/terraform.tfstate"
```

Comment:
@question: what is the environment variable describing? is it the region? or the environment name? how would the environment differ from the project_name?

Reply:
Environment now optional,
Description provided.